### PR TITLE
feat(AS-5759): amendment to fix save and return …

### PR DESCRIPTION
**Pull request by guiribeirokc**
…to be able to continue

amendment to fix save and return to be able to continue

feat#AS-5759

## Ticket Number

https://pins-ds.atlassian.net/browse/AS-

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by chrisHowarthkc — DISMISSED** [review]

File: packages/appeals-service-api/src/lib/notify.js [hunk 1]
```diff
@@ -111,12 +112,12 @@ const sendSaveAndReturnContinueWithAppealEmail = async (appeal) => {
 			appeal.appealType,
 			appeal.eligibility.applicationDecision
 		);
+		const encrypted = CryptoJS.AES.encrypt(appeal.id, '');
```
[the remark this answers is not in the document]
Comment (guiribeirokc) [reply packages/appeals-service-api/src/lib/notify.js:115]: Yes you're right I've forgot to remove the unused encrypt. I can actually get the functionality and then we can discuss the need of encrypt later.

File: packages/business-rules/src/config.js [hunk 2]
```diff
@@ -175,12 +175,12 @@ const config = {
...
 						reference: appeal.id
 					}),
-					saveAndReturnContinueAppeal: (appeal, baseUrl, deadlineDate) => ({
+					saveAndReturnContinueAppeal: (appeal, encryptedId, baseUrl, deadlineDate) => ({
 						recipientEmail: appeal.email,
 						variables: {
 							'application number': appeal.planningApplicationNumber,
 							date: format(deadlineDate, 'dd MMMM yyyy'),
-							link: `${baseUrl}/full-appeal/submit-appeal/enter-code`
+							link: `${baseUrl}/appeal-householder-decision/enter-code/${encryptedId}`
```
Comment (PaulGruffyddAmaze) [inline packages/business-rules/src/config.js:183]: This appears to be using the same URL structure as the householder appeal above. Is that correct for this appeal type?

File: packages/appeals-service-api/package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
 		"body-parser": "^1.19.1",
 		"body-parser-csv": "^1.1.0",
 		"compression": "^1.7.4",
+		"crypto-js": "^4.1.1",
```
Comment (PaulGruffyddAmaze) [inline packages/appeals-service-api/package.json:32]: Is this still being used? If not, we should remove it.

**Comment by guiribeirokc** [comment]
The changes are here: packages/business-rules/src/config.js
We don't need to change the template

**Review by PaulGruffyddAmaze — APPROVED** [review]
I think that addresses all of my comments so gets a thumbs up from me.

**Pull request merged**